src/components/ControlsToggles/stories/Tabs.story.tsx

```diff
@@ -30,10 +32,10 @@ controlsTogglesStories.add('Tabs', () => (
         defaultSelected="item-2"
         borderColor={select('borderColor', colorOptionsWithTransparent, 'transparent')}
         backgroundColor={select('backgroundColor', colorOptionsWithTransparent, 'transparent')}
-        textColor={select('textColor', colorOptionsWithTransparent, colors.space100) as Colors[keyof Colors]}
+        textColor={select('textColor', colorOptionsWithTransparent, colors.space100) as Color}
```

Comment:
Pode tipar o retorno do select:
```suggestion
        textColor={select<Color>('textColor', colorOptionsWithTransparent, colors.space100)}
```
ref: https://github.com/storybookjs/storybook/blob/98c6bd9a499fd50a1df215db9b241c3a2d097f9c/addons/knobs/src/index.ts#L72-L85